src/platform/datapath_raw.h

```diff
@@ -78,6 +78,7 @@ typedef struct CXPLAT_DATAPATH {
 typedef struct CXPLAT_INTERFACE {
     CXPLAT_LIST_ENTRY Link;
     uint32_t IfIndex;
+    uint32_t ActualIfIndex;
```

Comment:
IMO we should do this the other way around: we should track each interface's `IfIndex` and also an optional `VfIfIndex`. We should then not have any VFs directly in the interface table, and only explicitly VF-aware code needs to use the `VfIfIndex`.

Reply:
And if we can't find the VF for an interface, we could just set VfIfIndex to the regular IfIndex value, or some magic value like `IFI_UNSPECIFIED`.

Reply:
Currently I'm not sure where the conventional IfIndex should be used and so use ActualIfIndex everywhere. 
If we use VfIfIndex as VF-aware code, there whould require if-else where code refers IfIndex